FreeAPS/Sources/Services/LiveActivity/LiveActivityBridge.swift

```diff
@@ -65,35 +65,44 @@ extension LiveActivityAttributes.ContentState {
             Self.formatGlucose(glucose - $0, mmol: mmol, forceSign: true)
         }) ?? ""
 
-        let chartBG = chart.map(\.glucose)
+        let detailedState: LiveActivityAttributes.ContentAdditionalState?
 
-        let conversionFactor: Double = settings.units == .mmolL ? 18.0 : 1.0
-        let convertedChartBG = chartBG.map { Double($0) / conversionFactor }
+        switch settings.lockScreenView {
+        case .detailed:
+            let chartBG = chart.map(\.glucose)
 
-        let chartDate = chart.map(\.date)
+            let conversionFactor: Double = settings.units == .mmolL ? 18.0 : 1.0
```

Comment:
Just noting that other parts of Trio use:
```
let conversionFactor = 0.0555
```

Whereas this code is using (not quite) the inverse of that. It may be out of scope of this PR, but units conversion should be handled consistently.

Reply:
Was this fixed?

Reply:
not in this commit yet.

Reply:
@bjornoleh it's out of scope for his PR, as also mentioned by @marionbarker in her testing comment. hence the resolution :)

Reply:
Ok, if the divide by 18 was not introduced here, then I agree. If it was, then I suppose it should be fixed. 

Not sure if it would matter for the end result, but there are rounding issues when comparing CGM values between other apps, hopefully we are not introducing differences in rounding within this app, as this can throw people onto the troubleshooting wagon for no reason (that’s my main concern).

Reply:
@bjornoleh please take a peek at the git diff in the PR. This was not introduced but only moved (from line 70-71 to now inside a `switch case`). 
![image](https://github.com/nightscout/Trio/assets/48965855/984052af-472b-4292-bf5a-bfa6e6aa8d61)

Reply:
Perfect, thanks, that’s all I needed to know, it was not so easily understood from looking at it on the phone (have probably missed this being pointed out before too). I’m not giving a formal review now, but I have no objections to merging.